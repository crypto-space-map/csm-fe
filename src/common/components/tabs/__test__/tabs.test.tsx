import { render } from '@testing-library/react';

import { Tab, Tabs } from '..';

describe('<Tabs /> ans <Tab />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <Tabs value="snapshot">
        <Tab value="snapshot" label="ISnapshot" />
      </Tabs>
    );
    expect(container).toMatchSnapshot();
  });
  it('should be defined', () => {
    const { container } = render(
      <Tabs value="defined">
        <Tab value="defined" label="IDefined" />
      </Tabs>
    );
    expect(container).toBeDefined();
  });
});
