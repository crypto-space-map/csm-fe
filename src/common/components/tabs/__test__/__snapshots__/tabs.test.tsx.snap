// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tabs /> ans <Tab /> should match snapshot 1`] = `
<div>
  <div
    class="MuiTabs-root css-1ujnqem-MuiTabs-root"
  >
    <div
      class="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
      style="overflow: hidden; margin-bottom: 0px;"
    >
      <div
        class="MuiTabs-flexContainer css-heg063-MuiTabs-flexContainer"
        role="tablist"
      >
        <button
          aria-selected="true"
          class="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-4dz997-MuiButtonBase-root-MuiTab-root"
          role="tab"
          tabindex="0"
          type="button"
        >
          ISnapshot
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
      <span
        class="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
        style="left: 0px; width: 0px;"
      >
        <span
          class="css-10hb3jh"
        />
      </span>
    </div>
  </div>
</div>
`;
